.gitignore

```diff
@@ -2,4 +2,6 @@
 node_modules
 
 # local dev notes
-NOTES.md 
\ No newline at end of file
+NOTES.md 
+
+**/.DS_Store
```

Comment:
.DS_Store is a useless file that keeps regenerating as far as I'm aware. It would be good to keep this in the `gitignore` so it doesn't keep reappearing and causing issues, especially with any merge conflicts.

Reply:
Fair enough!